docs/modules/setup/pages/install.adoc

```diff
@@ -13,157 +15,62 @@
-:uri-k8s-publish-service: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
-:uri-k8s-definition-files: https://raw.githubusercontent.com/yuzutech/kroki/master/docs/modules/setup/examples/k8s
-:uri-localhost-8000: pass:q[\http://localhost:**8000**]
 
-To convert plain text diagrams to images with Kroki,
-you first need to install the gateway server.
-
-[TIP]
-====
-We recommend installing Kroki using containers with either *Docker* or *Podman*, especially if you don't want to manually install diagrams libraries.
-Read more about choosing Docker or Podman at xref:about-docker-podman.adoc[].
-
-However, you can also install each diagrams libraries that you want to use and run the gateway server as a _standalone executable jar_.
-Consult the xref:manual-install.adoc[manual install page] to find out how.
-====
+This section covers installing Kroki locally.
 
 On this page, you'll learn:
 
-* [x] How to run Kroki
-* [x] How to configure Kroki
-
-== Install the Kroki gateway server
-
-Installing and running the Kroki gateway server by itself gets you started quickly to begin creating diagrams provided by the <<yuzutech-kroki,`yuzutech/kroki`>> image.
-
-Assumptions:
-
-* [x] {uri-docker-get-started}[Docker] or {uri-podman-get-started}[Podman] is installed on your machine.
-
-To start the Kroki gateway server, run the command:
-
-Using docker::
-
-[source,docker-cli]
-docker run yuzutech/kroki
-
-Using podman::
-
-[source,podman-cli]
-podman run yuzutech/kroki
-
-[IMPORTANT]
-====
-By default, when you start a container, it does not publish any of its ports to the outside world. +
-To make a port available to services outside of Docker or Podman, you need to use the `--publish` or `-p` flag.
-====
-
-[discrete]
-=== Published port
-
-To map a container port to a port on your Docker host, you need to use the `--publish` or `-p` flag.
-
-Using docker::
-
-[source,docker-cli]
-docker run -p8000:8000 yuzutech/kroki
-
-Using podman::
+//* [x] Why install Kroki locally
+* [x] How to get started
+* [x] The different ways to install Kroki
+* [x] About the Kroki pre-built image containers
 
-[source,podman-cli]
-podman run -p8000:8000 yuzutech/kroki
+//== Why install Kroki locally
 
-After the above command, the server will be available on your host at: `{uri-localhost-8000}`
+== Get started
+Use Docker or Podman as the easiest and fastest method to get started running Kroki locally on a system with internet access using the pre-built Kroki containers.
 
-[NOTE]
-====
-You can also map a different TCP port on the Docker host.
-For instance, if you want to map your host's TCP port 1234 to port 8000 in the Kroki container, use: `-p1234:8000`.
+.Minimal installation
+TIP: Regardless of your chosen containerized installation method, to get started and convert plain text diagrams to images with Kroki,
+you need only install the Kroki gateway server container "kroki"
```

Comment:
I'm not quite sure about this tip. 

Maybe we should tell that the gateway server provides quite a few diagram libraries (PlantUML, GraphViz, Vega...) with an internal cross-reference to `#yuzutech-kroki`.
And the tip would be: you don't have to start all the companion containers, you can only start/use the gateway server (to put emphasis on the fact that companion containers are *optional*).

What do you think?

Reply:
Agreed.

Actually, looking with fresh eyes, the Get Started section should not rely on tips to get started. This might be better.

== Get started
To get started, use Docker or Podman to start the mandatory kroki container only.

Use either <<Docker or Podman>> as the easiest and fastest method to get started running Kroki locally on a system with Internet access using the pre-built Kroki containers published on Docker Hub.

Start the mandatory <<#yuzutech-kroki,`yuzutech/kroki`>> container only to use a large number of diagram libraries immediately including PlantUML, GraphViz and more.
Depending on your diagram needs, this may be enough.

For the lists of diagram libraries pre-installed in each container see the section <<kroki-container-images>> on this page.

NOTE: Only the <<yuzutech-kroki>> container is mandatory.
All the companion containers are _optional_.